logconsumer_test.go

```diff
@@ -641,21 +641,47 @@ func Test_MultiContainerLogConsumer_CancelledContext(t *testing.T) {
 
 type FooLogConsumer struct {
```

Comment:
Given we are moving the `Hello from Docker!` assertion to the FooLogConsumer methods, I think we need to document it to avoid future mis-usages of that test struct. Do you think we can rename it to a meaningful name, such as `HelloWorldLogConsumer`?

```suggestion
// FooLogConsumer test log consumer that accepts logs from the "hello-world" Docker image, which
// prints out the "Hello from Docker!" log message.
type FooLogConsumer struct {
```